corehq/ex-submodules/casexml/apps/case/tests/util.py

```diff
@@ -231,16 +232,16 @@ def _delete_all(db, viewname):
                 pass
 
 
+@unit_testing_only
 def delete_all_cases():
-    assert settings.UNIT_TESTING
     _delete_all(CommCareCase.get_db(), 'case/get_lite')
 
 
+@unit_testing_only
 def delete_all_xforms():
-    assert settings.UNIT_TESTING
     _delete_all(XFormInstance.get_db(), 'couchforms/all_submissions_by_domain')
 
 
+@unit_testing_only
```

Comment:
fyi @dimagi/team-commcare-hq you can use this to explicitly only allow code to run in unit testing mode.